android/assets/jsons/translations/template.properties

```diff
@@ -726,6 +726,8 @@ Sleep until healed =
 Moving = 
 Set up = 
 Paradrop = 
+Add in capital = 
```

Comment:
I think the 'every action is translated' test is causing us to have fake translations... this text would never actually be shown, if I understand correctly

Reply:
Correct, I think this also happened before for the great engineer bonus iirc, where `Hurry Construction` is in the `template.properties` file, while only ever `Hurry Construction (+[amount]⚙)` is actually shown to the user